plugin.video.dailymotion_com/addon.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.dailymotion_com" name="DailyMotion.com" version="2.4.2+matrix.1" provider-name="gujal">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+		<import addon="script.module.six" version="0.0.1"/>
+		<import addon="script.module.kodi-six" version="0.0.1"/>
+        <import addon="script.module.requests" version="2.4.3"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">	    
+        <platform>all</platform>
+	    
+        <summary lang="en">Movies, TV, Music, User-Clips, and Livestreams</summary>
+		<summary lang="fr">Films, Séries TV, Musique, Clips, et Live</summary>    
+        <summary lang="he">סרטים, טלוויזיה, מוסיקה, סירטונים של המשתמש, תלת מימד והזרמה חיה</summary>   
+        
+		<description lang="en">Dailymotion attracts 300 million users from around the world, who watch 3.5 billion videos on its player each month.</description>
+		<description lang="fr">Dailymotion attire 300 millions d’utilisateurs qui regardent 3,5 milliards de vidéos sur son player chaque mois, à travers le monde.</description>
+		<description lang="he">Dailymotion מושך 300 מיליון משתמשים מרחבי העולם שצופים ב -3.5 מיליארד סרטונים מדי חודש.</description>
+	
+		<disclaimer lang="en">Some parts of this addon may not be legal in your country of residence - please check with your local laws before installing.</disclaimer>
+		<disclaimer lang="fr">Certaines parties de cet addon peuvent ne pas être légales dans votre pays de résidence - s’il vous plaît vérifier avec vos lois locales avant d’installer.</disclaimer>
+		<disclaimer lang="he">ייתכן שחלק מהתוספות האלו אינן חוקיות בארץ המגורים שלך - אנא בדוק את החוקים המקומיים שלך לפני ההתקנה.</disclaimer>
+	    
+        <news>
+[B]Version 2.4.2 - 15 July 2020[/B] 
+ - Rewrite with six library instead of future
+ - Fix playback issues
+		</news>
+		<website>https://www.dailymotion.com</website>
+        <source>https://github.com/Gujal00/plugin.video.dailymotion_com</source>
+        <forum>https://forum.kodi.tv/showthread.php?tid=142626</forum>
+        <email>gujal at protonmail dot com</email>
```

Comment:
license tag missing, please define it using the SPDX identifier of your license. See spdx.org for the list of identifiers

Reply:
Done as per advice